Framework/Algorithms/test/ReflectometrySumInQTest.h

```diff
@@ -243,7 +273,9 @@ class ReflectometrySumInQTest : public CxxTest::TestSuite {
   }
 
 private:
-  static Mantid::API::MatrixWorkspace_sptr testWorkspace() {
+  static Mantid::API::MatrixWorkspace_sptr
+  testWorkspace(const double centreTwoThetaDegrees = 0.87,
```

Comment:
This constant has a bad name. Either it is in radians or your conversion on L291 is wrong.

Reply:
I am sorry but I completely fail to see what is wrong with the conversion on L291. Could you elaborate?

Reply:
Line 291:
```C++
const double twoTheta{centreTwoThetaDegrees / 180. * M_PI};
```
suggests that `centreTwoThetaDegrees` is actually in radians.

Reply:
Conversion from radians to degrees would be `twoTheta / M_PI * 180.`, wouldn't it? Am I missing something here?

Reply:
`twoTheta` is in degrees. `centreTwoThetaDegrees` is in radians. The name, `centreTwoThetaDegrees` suggests a value that is in degrees.